src/interface/blas2_interface.hpp

```diff
@@ -79,8 +79,10 @@ typename sb_handle_t::event_t _gemv_impl(
   const auto x_vector_size = is_transposed ? _M : _N;
   const auto y_vector_size = is_transposed ? _N : _M;
 
-  auto mA = make_matrix_view<col_major>(_mA, _M, _N, _lda);
-  auto vx = make_vector_view(_vx, _incx, x_vector_size);
+  typename MatrixViewType<container_t0, index_t, col_major>::type mA =
```

Comment:
The changes to MatrixViewType<..> and VectorViewType<..> are necessary because we cannot replace them with `const auto`. The problem is that, in the case of buffer, VectorView/MatrixView cannot be `const` because of the `access_displacement`.